light_client/mvplightclient.py

```diff
@@ -61,19 +61,17 @@ def parseHexToByte(hex_string):
   #  ==========
   #  CHECKPOINT
   #  ==========
-  checkpoint_url = "https://api.allorigins.win/raw?url=https://lodestar-mainnet.chainsafe.io/eth/v1/beacon/states/head/finality_checkpoints" 
+  checkpoint_url = "https://lodestar-mainnet.chainsafe.io/eth/v1/beacon/states/finalized/finality_checkpoints"  
   checkpoint = callsAPI(checkpoint_url)
   finalized_checkpoint_root = checkpoint['data']['finalized']['root']  
-  print(checkpoint)
 
   #  =========
   #  SNAPSHOT
   #  =========
-  snapshot_url = "https://lodestar-mainnet.chainsafe.io/eth/v1/lightclient/snapshot/0xe7ec5a97896da6166bb56b89f9fcb426e13b620b1587dbedda258fd4faa00ab5" 
+  snapshot_url = "https://lodestar-mainnet.chainsafe.io/eth/v1/light_client/bootstrap/0x1669a323f2e9ddf8b918f959789428f1f5f588a8368b998ffc1d0d73d94d3f80" 

```

Comment:
I'm wanting for this checkpoint root to be inserted as a variable.  How do I do this in python?